lunamark/reader/markdown.lua

```diff
@@ -1126,6 +1126,9 @@ function M.new(writer, options)
   local function ordered_list(s,tight,startnum)
     if options.startnum then
       startnum = tonumber(startnum) or 1  -- fallback for '#'
```

Comment:
should this be `startNumber =` ?

Reply:
It is a good thing that I did not merge this right in. Sorry for not double-checking, `startnum` is `startNumber` in my fork.